airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/user_fields.json

```diff
@@ -0,0 +1,113 @@
+{
+  "properties": {
```

Comment:
Please update the schema following the reference here: https://docs.airbyte.com/connector-development/schema-reference